dlotter/read.py

```diff
@@ -399,7 +399,12 @@ def read(self, args:argparse.Namespace, files_to_read:list) -> xr.Dataset:
 
             f = nc.Dataset(files_to_read[k])
 
-            forecast = f.getncattr('ValidDate')
+            if not 'ValidDate' in f.ncattrs():
+                print('Warning: ValidDate not in attributes, setting date to Jan 1st 1900 UTC 12.') #If more than 12 files are used, this may cause trouble...
+                forecast = '1900-Jan-01 '+str(12+k)+':00:00'
+            else:
+                forecast = f.getncattr('ValidDate')
+
```

Comment:
To prevent a clash when using many files, we could use a dummy starttime before the loop and use an iterator to update the forecasttime.

maybe something like:

```suggestion
dummy_time = dt.datetime(1900,1,1,0)
for k,f in enumerate(files_to_read):
    f = nc.Dataset(files_to_read[k])
    if not 'ValidDate' in f.ncattrs():
        dummy_time+=dt.timedelta(minutes=k)
        print('Warning: ValidDate not in attributes, setting date to {}'.format(dummy_time.strftime('%Y-%b-%d %H:%M:%S')))
        forecast = dummy_time.strftime('%Y-%b-%d %H:%M:%S')
    else:
        forecast = f.getncattr('ValidDate')
```

Reply:
Beautiful, but using k makes the time increase in a strange sequence, I think just using 1 minute for each file is "cleaner"?

Reply:
I actually think we should give time as `NaN` if it is not found in the attributes. That will be much cleaner for the user.